.github/workflows/lint.yml

```diff
@@ -0,0 +1,19 @@
+name: Linting
```

Comment:
Might be good to add a section on the contributing.md on how to run this locally

Reply:
Yes, good point. Fixed.